pkgs/by-name/la/lace/package.nix

```
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lace";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "trolando";
+    repo = pname;
+    rev = "v${version}";
```

... To use `finalAttrs` here:

```suggestion
    rev = "v${finalAttrs.version}";
```